lib/WeBWorK/lib/WeBWorK/PG/Local.pm

```diff
@@ -296,226 +313,236 @@ sub new_helper {
-
-	############################################################################
-	# write timing log entry -- the translator is now all set up
-	############################################################################
-# 	writeTimingLogEntry($ce, "WeBWorK::PG::new",
-# 		"initialized",
-# 		"intermediate");
+            answers  => {},
+            result   => {},
+            state    => {},
+            errors   => "Failed to read the problem source file.",
+            warnings => "$warnings",
+            flags    => { error_flag => 1 },
+            pgcore   => $translator->{rh_pgcore},
+        }, $class;
+    }
 
-	############################################################################
-	# translate the PG source into text
-	############################################################################
+    ############################################################################
+    # install a safety filter
+    # FIXME -- I believe that since MathObjects this is no longer operational
+    ############################################################################
+    #warn "PG: installing a safety filter\n";
+    #$translator->rf_safety_filter(\&oldSafetyFilter);
+    $translator->rf_safety_filter( \&WeBWorK::PG::nullSafetyFilter );
 
-	#warn "PG: translating the PG source into text\n";
-	$translator->translate();
+    ############################################################################
+    # write timing log entry -- the translator is now all set up
+    ############################################################################
+    # 	writeTimingLogEntry($ce, "WeBWorK::PG::new",
+    # 		"initialized",
+    # 		"intermediate");
 
-	############################################################################
-	# !!!!!!!! IMPORTANT: $envir shouldn't be trusted after problem code runs!
-	############################################################################
+    ############################################################################
+    # translate the PG source into text
+    ############################################################################
 
-	my ($result, $state); # we'll need these on the other side of the if block!
-	if ($translationOptions->{processAnswers}) {
+    #warn "PG: translating the PG source into text\n";
+    $translator->translate();
 
-		############################################################################
-		# process student answers
-		############################################################################
+    ############################################################################
+    # !!!!!!!! IMPORTANT: $envir shouldn't be trusted after problem code runs!
+    ############################################################################
 
-		#warn "PG: processing student answers\n";
-		$translator->process_answers($formFields);
+    my ( $result, $state )
+      ;    # we'll need these on the other side of the if block!
```

Comment:
Why is this comment moved to a new line?

Reply:
accidental